dashboards/sql/tests/benefits_tab_queries/reproduce_issue.sh

```
@@ -0,0 +1,217 @@
+  tmp="$(mktemp)"
+  {
+    echo "SELECT set_config('app.white_label_id', '${WL_ID}', false);"
+    echo "EXPLAIN (FORMAT TEXT)"
+    cat "$file"
+  } >"$tmp"
+  psql -v ON_ERROR_STOP=1 -q -f "$tmp" >"${PLAN_DIR}/${STAMP}_${name}.txt" 2>&1 || true
+  rm -f "$tmp"
+}
+
+run_timed() {
+  local label="$1" sql_file="$2"
```

Shellcheck (0.11.0)

[warning] label appears unused. Verify use (or export if used externally).

(SC2034)